cli/tests/integrations/test_ssl.py

```diff
@@ -0,0 +1,26 @@
+from dcos import http, util
+from dcos.errors import DCOSException
+
+import pytest
+from six.moves import urllib
+
+DCOS_URL = util.get_config()['core.dcos_url'].replace('http://', 'https://')
+DCOS_URL = urllib.parse.urljoin(DCOS_URL, 'marathon/')
+
+
+def test_dont_verify_ssl():
+    resp = http.request("get", DCOS_URL, verify=False)
```

Comment:
Why are these integration tests calling the `http` module? Why aren't we using the CLI to test this stuff? I am interested in the UX. What do we print on success and on failure?